src/main/java/com/example/hop_oasis/service/data/RatingServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package com.example.hop_oasis.service.data;
+
+import com.example.hop_oasis.dto.ItemRatingDto;
+import com.example.hop_oasis.model.ItemType;
+import com.example.hop_oasis.model.Rating;
+import com.example.hop_oasis.repository.RatingRepository;
+import com.example.hop_oasis.service.RatingService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class RatingServiceImpl implements RatingService {
+    private final RatingRepository ratingRepository;
+
+    @Override
+    public double getAverageRating(Long itemId, ItemType itemType){
+        List<Rating> ratings = ratingRepository.findByItemIdAndItemType(itemId, itemType);
+        return ratings.stream()
+                .mapToDouble(Rating::getRating)
```

Comment:
this is what I said above. 
`Rating::getRating`
this doesn't make sense. Why would  you get rating if you already have ranking?))

Reply:
done